File: app/src/main/java/im/point/dotty/network/LogoutReply.java
package im.point.dotty.network;

import com.google.gson.annotations.SerializedName;

public final class LogoutReply extends Envelope {
    @SerializedName("ok") boolean isOk;

    public LogoutReply(boolean isOk) {
        this.isOk = isOk;
    }

    public boolean getOk() {
        return isOk;
    }
}

File: app/src/main/java/im/point/dotty/network/LogoutReply.kt
/*
 * Copyright (c) 2019-2021 Pavel A. Sokolov
 */
package im.point.dotty.network

import com.google.gson.annotations.SerializedName

data class LogoutReply(@SerializedName("ok")
                       var ok: Boolean? = null) : Envelope()